etcdmain/config.go

```diff
@@ -155,6 +155,7 @@ func newConfig() *config {
 		plog.Panicf("unexpected error setting up clusterStateFlag: %v", err)
 	}
 	fs.BoolVar(&cfg.StrictReconfigCheck, "strict-reconfig-check", cfg.StrictReconfigCheck, "Reject reconfiguration requests that would cause quorum loss.")
+	fs.BoolVar(&cfg.EnableV2Endpoints, "enable-v2-endpoints", true, "etcd will start the v2 endpoints for the clients.")
```

Comment:
s/enable-v2-endpoints/enable-v2/
s/etcd will start the v2 endpoints for the clients/Accept etcd V2 client requests.
?